### ISSUE TITLE
FSU Baseball/Softball StartingShot

### ISSUE THREAD

**Issue by adavila17** [Aubron/scoreshots-templates#60]
![fsu_baseball_startingshot](https://user-images.githubusercontent.com/26416133/29924259-6d3d0c48-8e2a-11e7-83fc-18b465dd6f17.jpg)

**Comment by adavila17** [comment]
I think we're good to just share this now? @lisajohnson11 what do you think? Although this could spur up more templates.....

**Comment by lisajohnson11** [comment]
Yeah I'm sure it will. We should probably re-visit that last tomorrow and see what else we could at least send over a proof of before I shoot this off.
